src/Pyz/Yves/Log/LogDependencyProvider.php

```diff
@@ -28,15 +28,23 @@ class LogDependencyProvider extends SprykerLogDependencyProvider
     protected function addLogHandlers(Container $container): Container
     {
         $container->set(static::LOG_HANDLERS, function () {
-            return [
-                new StreamHandlerPlugin(),
-                new ExceptionStreamHandlerPlugin(),
-            ];
+            return $this->getLogHandlers();
```

Comment:
The same question here and for the `addProcessors`. Do we need it on the project level? Already present in core module https://github.com/spryker/log/blob/2e35e47ce44b133dc197f43c3ab4b6a78d7ed6ae/src/Spryker/Zed/Log/LogDependencyProvider.php#L149-L153 and https://github.com/spryker/log/blob/2e35e47ce44b133dc197f43c3ab4b6a78d7ed6ae/src/Spryker/Zed/Log/LogDependencyProvider.php#L170-L175 respectively.